mealie/schema/user/user.py

```diff
@@ -76,7 +76,7 @@ def getter_dict(cls, name_orm: User):
         schema_extra = {
             "username": "ChangeMe",
             "fullName": "Change Me",
-            "email": "changeme@email.com",
+            "email": settings.DEFAULT_EMAIL,
             "group": settings.DEFAULT_GROUP,
             "admin": "false",
```

Comment:
This would expose the default email to anyone who has access to the API docs. We should leave it hardcoded.